engine/installation/linux/docker-ce/gentoolinux.md

```diff
@@ -3,6 +3,7 @@ description: Installation instructions for Docker on Gentoo.
 keywords: gentoo linux, docker, documentation,  installation
 redirect_from:
 - /engine/installation/gentoolinux/
+- /engine/installtion/linux/gentoolinux/
```

Comment:
Same here as for the Crux Linux page; I don't see a compelling reason to include this in the docs, because we have never shipped packages for this distro